src/menu_items.c

```diff
@@ -5690,25 +5912,22 @@ void add_menu_item(s32 type, s32 column, s32 row, s8 priority) {
         case MENU_ITEM_UI_START_BACKGROUND:
             // load_menu_img_comp_type(gMenuTexturesBackground[has_unlocked_extra_mode()], LOAD_MENU_IMG_TKMK00_ONCE);
             load_texture_reset_cache(gMenuTexturesBackground[has_unlocked_extra_mode()]->textureData);
+            resize_menu_texture(gMenuTexturesBackground[has_unlocked_extra_mode()]);
             break;
         case MENU_ITEM_UI_LOGO_AND_COPYRIGHT:
             load_mario_kart_64_logo();
             // sMenuTextureListIndex += 1;
             load_menu_img(seg2_copyright_1996_texture);
             break;
         case MENU_ITEM_UI_PUSH_START_BUTTON:
+            resize_menu_texture(seg2_push_start_button_texture);
             load_menu_img(seg2_push_start_button_texture);
             break;
         case MAIN_MENU_BACKGROUND:
         case CHARACTER_SELECT_BACKGROUND:
         case COURSE_SELECT_BACKGROUND:
-            load_texture_reset_cache(gMenuTexturesBackground[has_unlocked_extra_mode()]->textureData);
```

Comment:
This section is a regresttion. And more regressions under it.

Reply:
no longer use sMenuTextureList or sMenuTextureMap to render so it's useless